packages/ui/src/1_atoms/Paragraph/Paragraph.types.ts

```diff
@@ -0,0 +1,15 @@
+export enum ParagraphType {
+  regular = 'regular',
```

Comment:
Why do you have only `regular` type here if you have also the `light` and `bold` styles? Either you should add them here as well or delete them from styles (I am not sure what is correct because as I mentioned earlier, I wasn't able to find all of the variations in the design).

Reply:
updated, waiting for Gilad approve

Reply:
Hm, I dont even think that story requirements should ask paragraph to have style for light/bold.
From HTML side `p` should have only regular font and then nested with `strong`, `em`, etc if needed.

Reply:
@creed-victor I've removed type now and shifted the default font-weight style to base tailwind css styling. Normal weight can be set via css className (added examples to storybook for this)